src/KnobManager.ts

```diff
@@ -1,8 +1,9 @@
 /* eslint no-underscore-dangle: 0 */
 
-import { navigator } from 'global';
+// navigator exists in Jest but not the browser. global exists in the browser but not Jest
```

Comment:
This really confused me. If anyone knows the reason for this, please enlighten me!